pkg/controller/podgroup_test.go

```diff
@@ -128,6 +128,15 @@ func Test_Run(t *testing.T) {
 			previousPhase:     v1alpha1.PodGroupPending,
 			desiredGroupPhase: v1alpha1.PodGroupPreScheduling,
 		},
+		{
+			name:              "Pod number scale down to 0",
```

Comment:
Group status convert from running to pending ?

Reply:
Yes, if the associated Pods is always 0, the PodGroup will stay in Pending phase:
```go
	switch pgCopy.Status.Phase {
	case "":
		pgCopy.Status.Phase = schedv1alpha1.PodGroupPending
	case schedv1alpha1.PodGroupPending:
		if len(pods) >= int(pg.Spec.MinMember) {
			pgCopy.Status.Phase = schedv1alpha1.PodGroupPreScheduling
			fillOccupiedObj(pg, pods[0])
		}
	default:
...
```

I think the only situation that may cause problem is when associated Pods scale down to 0, which means PodGroup will convert Running to Pending.

Reply:
done